src/benchmarks/gc/src/exec/run_single_test.py

```diff
@@ -306,27 +306,29 @@ def _get_perfview_start_or_stop_cmd(
 
 _DEFAULT_MAX_TRACE_SIZE_GB = 1
 
+_collect_options = {
+    CollectKind.gc: ["-GCCollectOnly"],
+    CollectKind.verbose: ["-GCCollectOnly", "-ClrEventLevel:Verbose"],
+    # Need default kernel events to get the process name
+    CollectKind.cpu_samples: [
+        "-OnlyProviders:ClrPrivate:1:5,Clr:1:5",
+        "-ClrEvents:GC+Stack",
+        "-ClrEventLevel:Verbose",
+        "-KernelEvents:Default",
+    ],
+    CollectKind.thread_times: [
+        "-OnlyProviders:ClrPrivate:1:5,Clr:1:5",
+        "-ClrEvents:GC+Stack",
+        "-ClrEventLevel:Verbose",
+        "-KernelEvents:Default,ThreadTime",
```

Comment:
>"-KernelEvents:Default,ThreadTime" [](start = 8, length = 34)

I don't think you need both Default and ThreadTime, ThreadTime alone is sufficient

Reply:
Are we certain about this? I left it that way since Andy originally had it as `Default,ContextSwitch`. How can I know whether default events were recorded? I ask to run tests and compare to make sure we can leave just `ThreadTime`.

Reply:
see [KernelTraceEventParser.cs#L178](https://github.com/microsoft/perfview/blob/master/src/TraceEvent/Parsers/KernelTraceEventParser.cs#L178). feel free to run tests and collect with both cmdline args to make sure.

Reply:
Thanks for pointing me out to the docs, Maoni. I also ran a couple tests and indeed we seem to have the same behavior. I have now removed those parameters.